apps/condo/domains/common/hooks/useAudio.tsx

```diff
@@ -0,0 +1,55 @@
+import getConfig from 'next/config'
+import { useEffect, useRef } from 'react'
+
+const {
+    publicRuntimeConfig,
+} = getConfig()
+
+const { audioConfig } = publicRuntimeConfig
+
+interface IUseAudio {
+    playNewItemsFetchedSound: () => void
+}
+
+let newItemsAudio
+if (typeof window !== 'undefined') {
+    if (audioConfig.newItemsAudioPath) {
+        newItemsAudio = new Audio(audioConfig.newItemsAudioPath)
+    } else {
+        console.warn('Missing or incorrect AUDIO_CONFIG. Notification sounds will not be available')
+    }
+}
+
+const warnOnMissingAudioConfig = () => {
+    console.warn('Could not play sound, because AUDIO_CONFIG is missing or incorrect')
+}
+
+export const useAudio = (): IUseAudio => {
+    const isTabTouched = useRef(false)
+
+    const playNewItemsFetchedSound = () => {
+        // TODO(antonal): Find what permissions are not allowing to play audio in script without user action
```

Comment:
If you have not solved the problem, then you should create a task and write in todo so as not to forget